### PR TITLE
New Targets 

### PR DESCRIPTION
**Pull request by satyagupta**
Added new targets which are based on DIY 2400 JR module. And increased power level at 250mW to 0 (from -1) since now we are getting 220mW if we set 250mW in lua.

### CODE REVIEW

**Comment by JyeSmith** [comment]
I don't think this PR should be merged.

All e28 module will vary in output.  Some will output ~350mW on the 0 dBm setting.  So we will end up with another half a dozen targets all the same but with tweaked outputs for individual users.  This will start to clutter the target list even more for general users.

Manufacturers also have this issue, and an average setting needs to be used.

I think anyone at the level of measuring their module output can tweak the output power locally.

**Comment by satyagupta** [comment]
Hey Jye,

I am little confused here, we had a discussion in the ELRS Discorrd and this was the problem that we were facing.

> I have one question, i am able to see 10/25/50 mW power correctly
> but 100mW shows little less (97-98) and when i set 250mW it shows upto 220mW
> is that normal?

And you suggested that we should try changing the max power setting to 0 
https://discord.com/channels/596350022191415318/798006228450017290/871676790102446110

BTW we are not using e28, when I say based in DIY i mean its based on the schematics but not same tx module. We are using SX1280 and SKY85321 as amp. 
image

Anyways, are you suggesting that we should not increase the max power setting to 0 (keep it as it is) and try to let users change by themselves? 

Reverting back the powerlib changes would allow the merge?

**Comment by satyagupta** [comment]
Hi @JyeSmith 

We are making this for sales purpose, we would be selling this only in India to begin with. We would also be coming up with 2 more modules one for Slim and one for T-Lite.

We are also making nano receivers which are based directly on diy nano receiver and planning to get a diversity but with ESP8285 directly soldered on pcb instead of a module. 

Let me know if any more details are required.

**Comment by satyagupta** [comment]
FYI, i have changed target names to change format as others.

File: src/targets/quadkopters_2400.ini [hunk 1]
```diff
@@ -0,0 +1,31 @@
+
+# ********************************
+# Transmitter targets
+# ********************************
+
+[env:QuadKoters_JR_2400_TX_via_UART]
```
Comment (maybenikhil) [inline src/targets/quadkopters_2400.ini:6]: should this not be QuadKopters not QuadKoters?

**Review by maybenikhil — CHANGES_REQUESTED** [review]
spelling

File: src/targets/quadkopters_2400.ini (hunk 1 above)
Comment (satyagupta) [reply src/targets/quadkopters_2400.ini]: Yea thanks for pointing this out *embarrassed*

**Comment by satyagupta** [comment]
This is fixed and pushed.

**Pull request merged**